esp/services/esdl_svc_engine/esdl_binding.cpp

```diff
@@ -2274,6 +2274,45 @@ int EsdlBindingImpl::getQualifiedNames(IEspContext& ctx, MethodInfoArray & metho
     return methods.ordinality();
 }
 
+int EsdlBindingImpl::getMethodProperty(IEspContext &context, const char *serv, const char *method, StringBuffer &page, const char *propname, const char *dfault)
+{
+    if (!serv || !*serv || !method || !*method)
+        return 0;
+
+    if (!m_esdl)
+    {
+        ESPLOG(LogMax,"EsdlBindingImpl::getMethodDescription - ESDL definition for service not loaded");
```

Comment:
Should this be getMethodProperty instead of getMethodDescription? Same question applies to other ESPLOG statements.

Reply:
Yes.  I refactored my fix without updating the log text.  Good catch.